hera_qm/tests/test_firstcal_metrics.py

```diff
@@ -229,7 +229,7 @@ def test_run_metrics_two_pols(self):
         # but you never know.
         two_pol_known_results = os.path.join(DATA_PATH, 'example_two_polarization_firstcal_results.hdf5')
         np.random.seed(0)
-        self.FC.run_metrics(std_cut=1.0)
+        self.FC.run_metrics(std_cut=.5)
```

Comment:
Is there a lot of flagged data in the input file here? Or should we be concerned this keyword had to change significantly for the known_output to work?

Reply:
To be fair the "known_input" was acquired by calling running the metrics by hand with fixed environment variables (like the seed though no real random things were happening). The version of the code produced the same results on a single polarization test file we had before. The fact that it is different now might just mean we should update the test file, but log exactly what it is that is changing.

Reply:
I changed the `std_cut` only because it matches the the default value and when I reran it from the command line to make the new example file that's what I used. I can easily put it back. It's arbitrary.

So, the original input data file was 100% flagged, which made no sense. I don't know how that happened. I modified it so that it was only flagged where the gains were identically equal to 1. 

Before I updated the two_pol_known_results, I tried running the unit tests using the previous example file and found that it only disagreed on the fields that I would expect disagreement given a proper accounting of flags.

Reply:
Thanks Josh. That is reassuring for me at least. I'll let @plaplant or @adampbeardsley weigh in for finality.